<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Big Pig Compressor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="initialScreen">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;h1&gt;Big Pig Compressor&lt;/h1&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="compressModeButton">
          <property name="text">
           <string>Compression</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decompressModeButton">
          <property name="text">
           <string>Decompression</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="compressScreen">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;h2&gt;Compression&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="compressPathLineEdit">
          <property name="placeholderText">
           <string>Path to file or directory...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="selectFileCompressButton">
            <property name="text">
             <string>Select File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectDirCompressButton">
            <property name="text">
             <string>Select Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="compressButton">
          <property name="text">
           <string>Compress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="decompressScreen">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;h2&gt;Decompression&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="decompressPathLineEdit">
          <property name="placeholderText">
           <string>Path to .bpig file...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFileDecompressButton">
          <property name="text">
           <string>Select File (.bpig)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decompressButton">
          <property name="text">
           <string>Decompress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton_2">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_backButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
